src/utils/redis.ts

```diff
@@ -9,6 +13,17 @@ export function createSimpleRedisClient(redisUrl?: string): redis.RedisClient {
   return redis.createClient(redisUrlOrDefault);
 }
 
+export const offuscateTokensInfo = (message: string) =>
```

Comment:
you probably can filter sensitive data application wide using an AI telemetry initializer

https://github.com/microsoft/ApplicationInsights-JS#telemetry-initializers

(of course it's not needed for this PR, just sharing infos :-)